api/v1/tetragon/sensors.proto

```diff
@@ -29,6 +29,19 @@ message ListSensorsResponse {
 
 message ListTracingPoliciesRequest { }
 
+enum TracingPolicyState {
+	// invalid state
+	INVALID = 0;
```

Comment:
I usually see `UNKNOWN` instead of `INVALID` being used as 0 in similar enums, but invalid seems ok to me too.

Reply:
do you think I should change this?

Reply:
yeah, I would say so, I think it's only a change in 1-2 commits?